wapitiCore/attack/mod_wapp.py

```diff
@@ -90,15 +91,27 @@ def attack(self):
             self.log_blue("---")
 
         for application_name in sorted(detected_applications, key=lambda x: x.lower()):
+            
+            versions = detected_applications[application_name]["versions"]
+            categories = detected_applications[application_name]["categories"]
+
             self.log_blue(
                 MSG_TECHNO_VERSIONED,
                 application_name,
-                detected_applications[application_name]["versions"]
+                versions
             )
             self.add_addition(
                 category=TECHNO_DETECTED,
                 level=LOW_LEVEL,
                 request=request,
                 info=json.dumps(detected_applications[application_name])
             )
+
+            if versions and "Web servers" in categories:
```

Comment:
If think it should only be `if "Web servers" in categories` as we probably want to use `WEB_SERVER_VERSIONED` even if the version is missing (?)

Reply:
I think it is a security issue only if we can determine the version in use since it is something that you can hide.

Reply:
I get it. Then it would make sense to have it in additional too (with or without version)